client-mobile/src/translations/EN.ts

```diff
@@ -14,6 +14,21 @@ export const EN: ITranslation = {
   Collab: "Collab",
   "Grocery items will appear here": "Grocery items will appear here",
   Recipe: "Recipe",
-  "Add grocery item": "Add grocery item",
+  "Add grocery items": "Add grocery items",
+  "Edit grocery items": "Edit grocery items",
   "Adding new shopping list...": "Adding new shopping list...",
+  "Updating grocery items...": "Updating grocery items...",
+  Name: "Name",
+  "e.g., onion": "e.g., onion",
+  "Unit shortcut": "Unit shortcut",
+  "pcs, etc.": "pcs, etc.",
+  "Type a name and unit to add your first grocery item.":
+    "Type a name and unit to add your first grocery item.",
+  ingredientNameWithUnit: "{{unit}} of {{name}}",
+  ingredientName: "{{name}}",
+  label: "{{label}}",
+  required: "*",
+  errorRequired: "{{name}} is required",
+  errorMaxLength: "Maximum {{length}} signs",
```

Comment:
Let's be more precise.
```suggestion
  errorMaxLength: "{{actualLength}}/{{length}} characters",
```